drivers/hmis/app/graphql/types/hmis_schema/organization_sort_option.rb

```diff
@@ -0,0 +1,18 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::OrganizationSortOption < Types::BaseEnum
+    description 'HUD Organization Sorting Options'
+    graphql_name 'OrganizationSortOption'
+
+    Hmis::Hud::Organization::SORT_OPTIONS.each do |opt|
+      value opt.to_s.upcase, value: opt
```

Comment:
If we move the direction into the `sort_order` above, this would need two options, one for ascending one for descending.  I think there will be places where it doesn't make sense to include both, something like most-recent enrollments is always desc, no one ever needs the oldest enrollments.